packages/augur-sdk/src/Augur.ts

```diff
@@ -151,9 +153,9 @@ export class Augur<TProvider extends Provider = Provider> {
     return Augur.connector.bindTo(f);
   }
 
-  public async on<T extends SubscriptionType>(eventName: SubscriptionEventName | string, callback: Callback): Promise<void> {
+  public async on(eventName: SubscriptionEventName | string, callback: Callback): Promise<void> {
     if (isSubscriptionEventName(eventName)) {
-      return Augur.connector.on(eventName, callback);
+      return Augur.connector.on(eventName, callback as Callback);
```

Comment:
Do you need to include `as Callback`? I thought specifying the type in the parameters would be enough.